### PR TITLE
Adds missing shortcuts to TileEditor. Keeps fill-tool selected after use.

### LINKED ISSUE #10918: TileMap bucket-fill usability
It would be very handy to:

a) Have a keyboard shortcut for the TileMap bucket-fill. Select and duplicate already 
    have one (Ctrl-B & Ctrl-D).

b) Keep the bucket-fill selected after using it. For this it would also be nice to have a keyboard shortcut for 
    "pick tile" to go back into the default editing mode.

**Edit:**
A keyboard shortcut for transpose tile would also be nice.

I would like to work on this issue and propose the following shortcuts:
- Bucket fill (G)
- Paint Tile (P)
- Transpose Tile (T)

Setting bucket fill to (G) instead of the more obvious (B) would prevent accidentally miss-pressing (B) while selecting with (CTRL-B). It also is a common bucket-fill-shortcut in painting programs like Aseprite.

### PR DESCRIPTION
**Pull request by ghost**
The following keyboard shortcuts are now available:
-Bucket fill (G)
-Paint Tile (P)
-Transpose Tile (T)

Fixes #10918

### CODE REVIEW

File: editor/plugins/tile_map_editor_plugin.cpp [hunk 1]
```diff
@@ -550,6 +558,7 @@ void TileMapEditor::_draw_fill_preview(int p_cell, const Point2i &p_point, bool
 	PoolVector<Vector2> points = _bucket_fill(p_point, false, true);
 	PoolVector<Vector2>::Read pr = points.read();
 	int len = points.size();
+	int time_after = OS::get_singleton()->get_ticks_msec();
```
Comment (akien-mga) [inline editor/plugins/tile_map_editor_plugin.cpp]: Leftover from debugging? Doesn't seem used anywhere.

Comment (kerskuchen) [reply editor/plugins/tile_map_editor_plugin.cpp]: Wat!? I can't remember putting that there as I did not time anything. How did this end up in my code?

I'll remove that one and try updating the pull request. Thank you for pointing it out!

**Comment by akien-mga** [comment]
You can use `git commit --amend` to edit your topmost commit, and then after checking that it's good with `git show`, force push to your remote branch to update the PR with `git push --force`.

**Comment by kerskuchen** [comment]
I could remove the unnecessary line and update the pull request. Thank you for the instructions, those really helped and worked like a charm! :+1:

**Pull request merged**